tools/cloudinit/extra

```diff
@@ -0,0 +1,101 @@
+#cloud-config
```

Comment:
Can we give this file a more specific name on what it is used for?

Reply:
These extra nodes, are meant to be general purpose - will be used for testing. It will be possible to introduce malicious node (e.g. very slow node), or extra test tools like NetworkViewer. Nevertheless I've added comment in mentioned file that explains the concept